### ISSUE TITLE
awc-operator backup not working

### ISSUE THREAD

**Issue by fritz0011** [ansible/tower-operator#1582]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx-operator/issues) for duplicates.
- [X] I understand that the AWX Operator is open source software provided for free and that I might not receive a timely response.

### Bug Summary

awx backup with a nonmanaged postgres (postgres  version 14)

### AWX Operator version

 2.6.0

### AWX version

23.2.0

### Kubernetes platform

kubernetes

### Kubernetes/Platform version

v1.25.10

### Modifications

no

### Steps to reproduce

- non  managed postgresDB
- ~~ postgres image 14"
========
awx-operator deployment

```
---
apiVersion: awx.ansible.com/v1beta1
kind: AWX
metadata:
  name: awx
  namespace: awx
spec:
  postgres_configuration_secret: pg-secret
  projects_persistence: true
  projects_storage_access_mode: ReadWriteOnce
  web_extra_volume_mounts: |
    - name: static-data
      mountPath: /var/lib/awx/projects/public
  extra_volumes: |
    - name: static-data
      persistentVolumeClaim:
        claimName:  static-data-pvc


```
awx-backup.yaml (pvc already created)

```
---
apiVersion: awx.ansible.com/v1beta1
kind: AWXBackup
metadata:
  name: awxbackup-2023-10-08
  namespace: awx
spec:
  deployment_name: awx
  clean_backup_on_delete: true
  backup_pvc: backupawx
  postgres_image_version: "14"
  no_log: false

```
```
 kubectl get po -n awx
NAME                                              READY   STATUS    RESTARTS       AGE
awx-operator-controller-manager-8dbbc7dc8-bf5pl   2/2     Running   2 (7h1m ago)   46h
awx-task-569c7d8ddf-rmvj4                         4/4     Running   0              46h
awx-web-6bb5f55869-r99r6                          3/3     Running   0              46h
postgres-14-5696dbf94b-9m4c6                      1/1     Running   0              88d

```

according to  controller-manager  logs

kubectl logs -f deployments/awx-operator-controller-manager -c awx-manager -n awx

```
TASK [backup : Get the current resource task pod information.] *****************
task path: /opt/ansible/roles/backup/tasks/postgres.yml:85
ok: [localhost] => {\"api_found\": true, \"changed\": false, \"resources\": []}
TASK [backup : Set the resource pod as a variable.] ****************************
task path: /opt/ansible/roles/backup/tasks/postgres.yml:98
ok: [localhost] => {\"ansible_facts\": {\"awx_task_pod\": {}}, \"changed\": false}
TASK [backup : Set the resource pod name as a variable.] ***********************
task path: /opt/ansible/roles/backup/tasks/postgres.yml:106
ok: [localhost] => {\"ansible_facts\": {\"awx_task_pod_name\": \"\"}, \"changed\": false}
TASK [backup : Precreate database partitions] **********************************
task path: /opt/ansible/roles/backup/tasks/postgres.yml:110\nfatal: [localhost]: FAILED! => {\"msg\": \"The conditional check ''Created partitions for' in result.stdout' failed. The error was: error while evaluating conditional ('Created partitions for' in result.stdout): 'dict object' has no attribute 'stdout'\"}\n\r\
```
so....
https://github.com/ansible/awx-operator/blob/devel/roles/backup/tasks/postgres.yml

```
- name: Get the current resource task pod information.
  k8s_info:
    api_version: v1
    kind: Pod
    namespace: '{{ ansible_operator_meta.namespace }}'
    label_selectors:
      - "app.kubernetes.io/name={{ deployment_name }}-task"
      - "app.kubernetes.io/managed-by={{ deployment_type }}-operator"
      - "app.kubernetes.io/component={{ deployment_type }}"
    field_selectors:
      - status.phase=Running
  register: awx_task_pod
```

```
kubectl get pods -l "app.kubernetes.io/managed-by=awx-operator" -l "app.kubernetes.io/component=awx" -l "app.kubernetes.io/name=awx-task" --field-selector status.phase=Running -n awx
NAME                        READY   STATUS    RESTARTS   AGE
awx-task-569c7d8ddf-rmvj4   4/4     Running   0          46h
```

### Expected results

perform backup

### Actual results

failed to  backup => se above details

**Comment by kurokobo** [comment]
Addressed in https://github.com/ansible/awx-operator/issues/1572 and https://github.com/ansible/awx-operator/pull/1573

The fix will be released with the next version.

**Comment by fritz0011** [comment]
fixed on 2.7.0 :)
